optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc

```diff
@@ -0,0 +1,253 @@
+[[development-guide]]
+= Development guide
+
+== Project structure
+
+The project is a multi-module Maven project.
+
+.Module dependency tree diagram
+image::modules.dot.svg[align="center"]
+
+At the bottom of the module tree there are the backend and frontend modules,
+which contain the application source code.
+
+The standalone module is an assembly module that combines backend and frontend into a single executable JAR file.
+
```

Comment:
...the back end...

Reply:
So
..combines _the_ back end and _the_ front end into...
article before both of them?

Reply:
Or
...combines the back end and front end module into...
?

Reply:
Yes!